docs/cqrs/pipeline/index.md

```diff
@@ -0,0 +1,138 @@
+# Pipeline
+
+The LeanCode CoreLibrary utilizes ASP.NET middlewares to create customized pipelines for handling requests and responses. This section intends to showcase the setup of a basic pipeline and explore its inner workings.
+
+## Packages
+
+| Package | Link | Application in section |
+| --- | ----------- | ----------- |
+| LeanCode.CQRS.AspNetCore | [![NuGet version (LeanCode.CQRS.AspNetCore)](https://img.shields.io/nuget/vpre/LeanCode.CQRS.AspNetCore.svg?style=flat-square&logo=nuget)](https://www.nuget.org/packages/LeanCode.CQRS.AspNetCore/8.0.2260-preview/) | Configuration |
+| LeanCode.CQRS.MassTransitRelay | [![NuGet version (LeanCode.CQRS.MassTransitRelay)](https://img.shields.io/nuget/vpre/LeanCode.CQRS.MassTransitRelay.svg?style=flat-square&logo=nuget)](https://www.nuget.org/packages/LeanCode.CQRS.MassTransitRelay/8.0.2260-preview/) | MassTransit related middlewares |
+
+## Configuration
+
+To establish LeanCode CoreLibrary pipelines, the [MapRemoteCqrs(...)] method from `LeanCode.CQRS.AspNetCore` package needs to be invoked after calling the `UseEndpoints(...)` method on the `IApplicationBuilder`. In the following example, [MapRemoteCqrs(...)] is configured to handle:
```

Comment:
This sentence feels awkward. Maybe sth like (rough scratch, nothing final):

CQRS objects can only be registered in the ASP.NET request pipeline via endpoint routing. To register use `IEndpointRouteBuilder.MapRemoteCqrs` extension methods (note that it's not the same as `UseEndpoints` as with the minimal startup you can call it directly). In `MapRemoteCqrs` you can configure the inner cqrs requests pipeline.

Also, I'd extend this sample to also include `IServceCollection.AddCQRS` method to make it self contained.